net.sonic_pi.SonicPi.json

```diff
@@ -123,8 +123,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://github.com/jackaudio/jack2/archive/v1.9.17.tar.gz",
-                    "sha256": "38f674bbc57852a8eb3d9faa1f96a0912d26f7d5df14c11005ad499c8ae352f2"
+                    "url": "https://github.com/jackaudio/jack2/archive/v1.9.22.tar.gz",
+                    "sha256": "1e42b9fc4ad7db7befd414d45ab2f8a159c0b30fcd6eee452be662298766a849"
```

Comment:
`jack_wait` that is relied upon has been removed in https://github.com/jackaudio/jack2/commit/564c710eefc760a5683b88b3ecb8a29100d5274e#diff-01eb6db751ea65e62bc79606f6cb8c728110fd84a6319ce227f29222c8c99ac0

This is why it doesn't start.

Reply:
Oh! That message about wayland was distracting me but has been also shown when it worked, it had nothing to do with that 🙄 Cool, thanks!